java/ql/src/semmle/code/xml/MavenPom.qll

```diff
@@ -365,6 +365,17 @@ class PomProperty extends PomElement {
   PomProperty() { getParent() instanceof PomProperties }
 }
 
+/**
+ * A repository block inside of a maven pom.
```

Comment:
```suggestion
 * An XML element representing any kind of repository declared inside of a Maven POM XML file.
```
To make it more clear that this does not only cover an `repositories` child element.